install Python bindings into same CryptoMiniSat-specific installation directory rather than in Python installation directory
author: Kenneth Hoste (HPC-UGent)
--- cryptominisat-5.0.1/python/CMakeLists.txt.orig	2016-12-20 14:29:51.756760945 +0100
+++ cryptominisat-5.0.1/python/CMakeLists.txt	2016-12-20 14:29:59.786688501 +0100
@@ -14,7 +14,7 @@
 
 add_custom_target(python_interface ALL DEPENDS ${OUTPUT}/timestamp)
 
-install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install --record files.txt)")
+install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install --prefix ${CMAKE_INSTALL_PREFIX} --record files.txt)")
 
 if (ENABLE_TESTING)
     add_test (NAME pytest
